tests/requirement/test_requirement_set.py

```
@@ -0,0 +1,230 @@
+\
+import logging
+from django.test import TestCase
+from django.utils import timezone
+import datetime
+
+from endoreg_db.models import (
+    RequirementSet,
+    RequirementSetType,
+    Patient,
```

Ruff (0.11.9)

`endoreg_db.models.Patient` imported but unused

Remove unused import

(F401)